regolith/tests/test_validators.py

```diff
@@ -0,0 +1,11 @@
+from regolith.validators import validate_schema
+from regolith.schemas import schemas
+from regolith.exemplars import exemplars
```

Comment:
If the exemplars are only used in the tests, then they should be in the test suite, not the package.